Libraries/Components/ScrollResponder.js

```diff
@@ -112,6 +113,15 @@ type State = {
 };
 type Event = Object;
 
+function isTagInstanceOfTextInput(tag) {
+  var instance = getInstanceFromNode(tag);
+  return instance && (
+    instance.constructor.displayName === 'AndroidTextInput' ||
```

Comment:
Not sure this is the best way to do instanceof in React... seems sketchy. @gaearon is there a better way?